docs/dcp2_system_design.rst

```diff
@@ -717,17 +759,43 @@ descriptors, one for metadata files and one for ``links.json`` files.
        The DSS adapter uses the DSS version of the metadata file,
        converting it to this restricted ISO syntax.
 
+   A staging area may only contain exactly one entity object with a given
+   ``entity_id``. This precludes more than one version of any entity or
+   assigning the same ``entity_id`` to entities of different types. 
+
+   The ``.remove`` suffix is used to request the removal of an entity. It can
+   only be used in staging areas that have the ``is_delta`` property set to
+   ``true``. If an object has this suffix, it must have a size of zero bytes.
```

Comment:
Is it important to have a size of zero bytes? Aka should this be a requirement or a suggestion

Reply:
Being strict now makes future spec enhancements easier, say, if we wanted to describe the reason for the deletion inside the deletion marker. We we allow content now, we'd have to deal with it in a later revision of the spec.

Reply:
Typically the reviewer should resolve their comments when they feel their concern was addressed sufficiently. But since Andrew took over as reviewer I'll resolve this now.